### PR TITLE
PERMANENTLY removes stocks

### PR DESCRIPTION
**Pull request by Bibby0110**
# General Documentation

### Intent of your Pull Request

Stocks have 2 states:

1. Worthless to the point that you may as well just pray to get credits from the admins
2. Bugs out and goes to max price giving cargo millions

### Why is this change good for the game?

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Removes stocks

### What should players be aware of when it comes to the changes your PR is implementing?
Stocks are gone

### What general grouping does this PR fall under? 
Cargo

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
No

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
No

# Changelog

:cl:  
rscdel: Stocks are gone for good
/:cl:

### CODE REVIEW

**Comment by JamieD1** [comment]
Rework dont remove.

**Comment by Bibby0110** [comment]
> Rework dont remove.

There's no point in reworking this.
It's rotten at the core.

**Review by JamieD1 — CHANGES_REQUESTED** [review]
Rework/change dont remove.

**Comment by Bibby0110** [comment]
> Rework/change dont remove.

There
Is
Nothing
To
Rework
The
Mechanic
Is
Flawed
At
A
Fundamental
Level

You
Would 
have
to
rip everything out
(Like I'm doing now)
and 
start
from
scratch

**Review by jcatjmeow — DISMISSED** [review]
This is an improvement.

**Comment by Moja10** [comment]
I agree with Gamer, there is almost nothing salvageable in its current form. The experiment really has failed, cargo shitting out money from meme stocks that go up and down at random isn't a balanced mechanic. Why earn money doing actual work like complex bounties, resources, acquiring and selling expensive items, when you can just get completely lucky from an RNG system and have your money go 3x - 10x up at complete random? Or the great experience of doing cargo labor to get initial capital just to have a random Cargo Tech put it all into a stocks that by complete RNG decided to hit a rock bottom in their prices, and you lose it all. 

Stock Market takes all the existing broken cargo money methods and amplifies them up to 11. Gas exports create a ridiculous amount of credits for gases that can have a ridiculous credit/mole ratio? What if the credits arbitrarily got multiplied by 3 because of RNG? 

Real stocks have a basis in real factors, competition, business practices, state of market, etc etc. The stock market has no basis in anything, and that WAS planned to be added... in 2016. When the PR was made to begin with by Goofball, who has since abandoned the idea for years. Just like everyone else has, for years, there has been 0 improvement to the stock mechanic and it's time for it to go.

**Comment by patpol4** [comment]
revert that last commit and do it with the map editor or else it will be broken like this

**Comment by patpol4** [comment]
Also, huh

**Comment by JamieD1** [comment]
> > Rework/change dont remove.
> 
> There
> Is
> Nothing
> To
> Rework
> The
> Mechanic
> Is
> Flawed
> At
> A
> Fundamental
> Level
> 
> You
> Would
> have
> to
> rip everything out
> (Like I'm doing now)
> and
> start
> from
> scratch

Its removed currently, So you have a month to work it from scratch.

**Comment by Bibby0110** [comment]
> > > Rework/change dont remove.
> > 
> > 
> > There
> > Is
> > Nothing
> > To
> > Rework
> > The
> > Mechanic
> > Is
> > Flawed
> > At
> > A
> > Fundamental
> > Level
> > You
> > Would
> > have
> > to
> > rip everything out
> > (Like I'm doing now)
> > and
> > start
> > from
> > scratch
> 
> Its removed currently, So you have a month to work it from scratch.

I'm not reworking stocks

It's simply not a mechanic that can be modelled in this and be fun at the same time

**Review by JamieD1 — CHANGES_REQUESTED** [review]
This PR was made out of spite to me over discord voice, with that reasoning im closing it.
